src/controllers/controller.line.js

```diff
@@ -194,6 +185,43 @@ module.exports = DatasetController.extend({
 		return values;
 	},
 
+	/**
+	 * @private
+	 */
+	_resolveLineElementOptions: function(element) {
+		var me = this;
+		var chart = me.chart;
+		var datasets = chart.data.datasets;
```

Comment:
`datasets` is used only in the line below, so I would alias `dataset` directly from `chart.data.datasets[me.index]` and leave this line out.

Reply:
Done